docs/contributing.rst

```diff
@@ -244,3 +244,24 @@ Use double underscore to organize tests into groups, for example:
 
    def test_foo__accepts_strings():
      ...
+
+PR/Git ops
+~~~~~~~~~~
+
+We currently use ``rebase`` or ``squash`` PR merge strategies. This means that
```

Comment:
Am I missing something or does this strategy not imply several specific changes to the dev workflow beyond general suggestions like keeping commits small so that they make more sense when rebased rather than merged into one?  I get that the global repo setting only applies to how PRs are merged now, but what happens if there is a merge commit in a branch for a PR?

Reply:
@eric-czech the answer to that is: that it depends. If you follow the recommendations from the links below and use rebase/squash you should not end with merge commits in your feature branch. If you feature branch nevertheless ends up with merge commit inside, `rebase` merge of that specific PR **might** (it's not guarantee) not be possible due to conflicts, and instead you either have to use `squash` merge (which would automatically squash all changes into a single commit for you), or locally rebase and push (as suggested by the official git docs).

Reply:
👍